liferay-frontend/liferay-portal
# [Only 7.4] - LPS-125104 - Remove compat-panels

This PR removes the compat layer for panels:

![image](https://user-images.githubusercontent.com/7963804/102611249-ddc66580-412e-11eb-919d-3d1abace085d.png)

We updated classes names and remove the compat CSS.

---

## To test the PR:

- Add an old portlet with old panels like: Message Boards
- Create a new Message board
- There you can see panels
